neuraxle/base.py

```diff
@@ -807,9 +861,36 @@ def handle_transform(self, data_container: DataContainer, context: ExecutionCont
         out = self.transform(data_container.data_inputs)
         data_container.set_data_inputs(out)
 
+        data_container = self.handle_after_any(data_container)
+
+        return data_container
+
+    def handle_after_any(self, data_container) -> DataContainer:
+        return self.hash_data_container(data_container)
+
+    def hash_data_container(self, data_container):
```

Comment:
the handle_after_any method was the idea I was talking to you about : have more handler methods to avoid duplicate code in steps that only require implementing some part of the handle methods i.e. after, or before...

Reply:
this would simplify a lot of code in the more complex steps that use handle methods.

Reply:
ill completely remove this idea because we don't have time to talk about it.